packages/data-stores/src/site/actions.ts

```diff
@@ -95,25 +95,36 @@ export function createActions( clientCreds: WpcomClientCredentials ) {
 		type: 'RESET_RECEIVE_NEW_SITE_FAILED' as const,
 	} );
 
-	const launchSiteStart = ( siteId: number ) => ( {
-		type: 'LAUNCH_SITE_START' as const,
+	const launchSiteIdle = ( siteId: number ) => ( {
+		type: 'LAUNCH_SITE_IDLE' as const,
 		siteId,
 	} );
 
-	const launchSiteComplete = ( siteId: number ) => ( {
-		type: 'LAUNCH_SITE_COMPLETE' as const,
+	const launchSiteSuccess = ( siteId: number ) => ( {
+		type: 'LAUNCH_SITE_SUCCESS' as const,
 		siteId,
 	} );
 
+	const launchSiteFailure = ( siteId: number, error: string ) => ( {
```

Comment:
Should the type for `error` be something like an `LaunchError` enum?

Reply:
Correct! I changed it.